tests/test_basic.py

```diff
@@ -314,7 +314,7 @@ def test_sorting_output(tmpdir, runner, create):
 
     examples = [
         ('-summary', ['aaa', 'bbb']),
-        ('due', ['aaa', 'bbb'])
+        ('due', ['aaa', 'bbb']),
```

Comment:
This change is irrelevant to this PR. There are a few other comma additions/removals, please exclude these (they just add noise to the diff/history).